src/main/java/org/jboss/JiraIssueLotteryCommand.java

```diff
@@ -49,10 +55,21 @@ private JiraConfiguration setupJiraConfiguration() {
         return jiraConfiguration;
     }
 
+    @Inject
+    ObjectMapper objectMapper;
+
     @Override
     public void run() {
         NewIssueCollector newIssueCollector = NewIssueCollector.getInstance(jiraEndpoint);
         Exchange exchange = jiraEndpoint.createExchange();
+        try {
+            LotteryConfig lotteryConfig = objectMapper.readValue(
+                    new URI(jiraLotteryAppConfig.configFileRepo().getRawContentsUrl()).toURL(),
+                    LotteryConfig.class);
+            Log.info(lotteryConfig);
```

Comment:
probably remove this later

Reply:
For sure, this is just temporary logging to see, what it does